src/util/FactorUtil.js

```diff
@@ -138,6 +157,7 @@ const factorData = {
     sortOrder: 17,
   },
   GENERIC_OIDC: {
+    /* eslint-disable-next-line @okta/okta/no-unlocalized-text */
     label: '',
```

Comment:
For some reason, I thought if we detected an empty string we didn't report. We should address this in a future PR into `okta-ui-configs`.